### PR TITLE
Setup Hello world Microverse repository and setup linters on the repo

### PR DESCRIPTION
**Pull request by r-ahmed2022**
Setup Hello world Microverse repository and setup linters on the repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
...
+
+#### Getting Started
+This is an example of how you may give instructions on setting up your project locally. Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.
+
+To get a local copy up and running follow these simple example steps.
+
+## Prerequisites
+
+Setup GitHub repository
+Install git
```
Comment (roshan-bajgain) [inline README.md]: - [ ] Kindly add a descriptive README file to your project. please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @r-ahmed2022 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight :tada: :champagne: 
- No linter error :heavy_check_mark:
- GitHub flow is followed :+1:


## Required Changes ♻️
- [x]  Pull Request should have a descriptive title telling you what changes are inside. So kindly fix it

__Check the comments under the review.__

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (r-ahmed2022) [reply README.md]: Thank you , Sure, will do as advised

**Comment by r-ahmed2022** [comment]
Done the required changes

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @r-ahmed2022 ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Nice commits messages✔️

## Required Changes ♻️
 - [ ] Like the previous reviewer said, your Pull Request should have a descriptive title telling you what changes you made, you can learn more about [the basic rules of a professional repo here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

 Alt text

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me with @danielufeli. in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by roshan-bajgain** [comment]
Hello @r-ahmed2022,
 your Pull Request should have a descriptive title telling you what changes you made. This means that you have to write the description here `see the below image` what you did. Thanks
pull request

**Comment by r-ahmed2022** [comment]
Updated the pull repository title to a more descriptive one as advised.
Thanks

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @r-ahmed2022 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966

## Highlights ⚡ 
✅ Linters are passing 👏 


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by r-ahmed2022** [comment]
feature branch merged to master branch as instructed in the Day exercise

**Pull request merged**